components/camel-disruptor/src/test/java/org/apache/camel/component/disruptor/SedaDisruptorCompareTest.java

```diff
@@ -232,7 +233,7 @@ void speedTestDisruptor(
         System.out.println("Starting real test of: " + componentName);
 
         forceGC();
-        Thread.sleep(1000);
+        Awaitility.await().atMost(1, TimeUnit.SECONDS);
```

Comment:
This is missing the actual check to performed. Probably you need to put `untilAsserted` or something like that.